dask_cuda/disk_io.py

```diff
@@ -169,12 +169,16 @@ def disk_write(path: str, frames: Iterable, shared_filesystem: bool, gds=False)
         import kvikio
 
         with kvikio.CuFile(path, "w") as f:
+            futs = []
             for frame, length in zip(frames, frame_lengths):
-                f.pwrite(buf=frame, count=length, file_offset=0, buf_offset=0).get()
+                futs.append(
+                    f.pwrite(buf=frame, count=length, file_offset=0, buf_offset=0)
+                )
```

Comment:
This could be replaced by a single vectorized write once available ( https://github.com/rapidsai/kvikio/issues/77 )

Reply:
To simplify a bit now, we could remove the `count=length, file_offset=0, buf_offset=0` arguments and create `futs` as a list comprehension. AFAICT, we don't not need  `frame_lengths` since we are always writing the full `frame`?

I think the same goes for the `pread` case.